papermill/iorw.py

```diff
@@ -26,10 +28,20 @@ def __init__(self):
         self.reset()
 
     def read(self, path):
+        if not fnmatch.fnmatch(os.path.basename(path), "*.*"):
```

Comment:
Do we need this test for "any kind of extension" now that there is the explicit test for `.ipynb` and `.json`?

Reply:
I think it's fine either way